common/src/main/java/org/apache/uniffle/common/config/RssConf.java

```diff
@@ -645,4 +646,8 @@ public String getEnv(String key) {
     return System.getenv(key);
   }
 
+  @VisibleForTesting
+  public Map<String, Object> getSettings() {
```

Comment:
Could we remove this method? It seems unused.

Reply:
It is used in `getShuffleAssignmentsTest`.

Reply:
We have already used reflection there, we don't need add this method, we can get this value by reflection.